velox/vector/FlatVector.h

```diff
@@ -403,6 +403,12 @@ class FlatVector final : public SimpleVector<T> {
     return nullptr;
   }
 
+  // Allocate buffer for not inline StringView from stringBuffers or pool,
+  // return the start pointer of allocated memory
+  char* getRawStringBufferWithSpace(vector_size_t /* unused */) {
```

Comment:
> /* unused */

Replace with parameter name, e.g. size. Unused parameter names should be commented out only in the implementation.